etc/profile-m-z/zoom.profile

```diff
@@ -20,14 +22,20 @@ whitelist ${HOME}/.cache/zoom
 whitelist ${HOME}/.config/zoomus.conf
 whitelist ${HOME}/.zoom
 include whitelist-common.inc
+include whitelist-var-common.inc
 
 caps.drop all
 netfilter
 nodvd
+nogroups
 nonewprivs
 noroot
 notv
+nou2f
```

Comment:
Does zoom support u2f?

Reply:
i don't think so. Only when you are using SSO, but it will open browser for auth then.